cohortextractor2/dsl.py

```diff
@@ -237,13 +246,18 @@ def __hash__(self) -> int:
         return hash(repr(self.value))
 
 
+@dataclass
+class Column:
+    name: str
+
+
 def not_null_patient_series(patient_series: PatientSeries) -> PatientSeries:
     comparator_value = Comparator(lhs=patient_series.value, operator="__ne__", rhs=None)
     return PatientSeries(value=comparator_value)
 
 
 def categorise(
-    mapping: dict[Expression, PatientSeries], default: Expression | None = None
+    mapping: Mapping[Expression, PatientSeries], default: Expression | None = None
```

Comment:
What does the switch to Mapping do here?

Reply:
See my answer to Peter's question.